test/unitTests/common.test.ts

```diff
@@ -72,24 +72,28 @@ suite("Common", () => {
         test("same paths", () => {
             let subfolder: string = ["C:", "temp", "VS", "dotnetProject"].join(path.sep);
             let folder: string= ["C:", "temp", "VS", "dotnetProject"].join(path.sep);
+            // tslint:disable-next-line:no-unused-expression
```

Comment:
Can this be done more generally? https://www.npmjs.com/package/tslint-no-unused-expression-chai perhaps?